fbw-a380x/src/systems/instruments/src/MFD/pages/common/MfdUiService.tsx

```diff
@@ -79,6 +87,7 @@ export class MfdUiService {
 
     const parsedUri = this.parseUri(nextUri);
     this.activeUri.set(parsedUri);
```

Comment:
Eventually, we could replace this with the publisher. Then we do not set this in two places.